aiida/storage/psql_dos/orm/querybuilder/main.py

```diff
@@ -367,7 +333,7 @@ def _build(self) -> Query:
             # LINK-PROJECTIONS #########################
 
             for vertex in self._data['path'][1:]:
-                edge_tag = vertex.get('edge_tag', None)  # type: ignore
+                edge_tag = vertex.get('edge_tag', None)
```

Comment:
@ltalirz you changed this, hence the failing mypy

Reply:
cheers just realized it. I still don't quite get why mypy locally and on ci often gives different results